runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -2211,6 +2238,7 @@ void init_ofiExchangeAvInfo(void) {
   size_t my_addr_len = 0;
 
   OFI_CHK_1(fi_getname(&ofi_rxEp->fid, NULL, &my_addr_len), -FI_ETOOSMALL);
+
```

Comment:
Is this related to the touch bar PR, or a side effect of looking at something else?